openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/workspace/sd_requirements.txt

```diff
@@ -0,0 +1,4 @@
+gdown==3.13.0
```

Comment:
is this file just a copy of the sd_requirements.txt  undex `envoy_workspace`?

Reply:
Yes, it is for model testing.

Reply:
Try to reuse Shard Descriptor and requirements from the Envoy's folder. So we can avoid repeated code